### ISSUE TITLE
Requested tag from Clinical Genetcs // 

### ISSUE THREAD

**Issue by KickiLagerstedt** [Clinical-Genomics/scout#605]
Regarding "Clinical report" Clinical Genetics would like to have the following options regarding variants (new)

Too common in public databases
Too common in local database
Not relevant phenotype
Inconsistent inheritance pattern
No compound variant
Not in RefSeq transcript
Does not affect splicing

WE would like to have the opportunity to check several options (krysslista motsvarande ex paneler)
![bild](https://user-images.githubusercontent.com/17139607/30274155-6fd33876-96fc-11e7-85ef-d55d6915cd68.png)

If any chosen we would like to see this as a grey dot (maybe) on the variants page.

**Comment by dnil** [comment]
:+1: I think this is perfect for a report function: really helps with audit trace for us. 
A very easy way would be to use that as a controlled vocabulary "phrase"-list that can be stored as comments. 
A second option would be a new dictionary to help with reporting. Variants with any kind of annotation e.g one of these, a pin, ACMG, variant tag etc would be exported. The nice thing about this is that they would also indicate what variant annotations to include on the report. If too common, frequency is nice on the report. If wrong phenotype, include OMIM diseases for the gene.

**Comment by moonso** [comment]
Hi, would it be possible to use the `variant tag` field for this? We could customise it for your needs
![skarmavbild 2017-09-12 kl 09 11 43](https://user-images.githubusercontent.com/405278/30312622-92cb6f12-979a-11e7-94b8-c157e6887cd8.png)

**Comment by dnil** [comment]
As long as we allow multiple values that should be fine. But keep in mind that we currently also use the variant tag as the only classification for SVs; and CMMS still I believe for additional info in addition to ACMG.
So at least some of the present variant tags should still be on there. Consider including as set phrases for comments - that immediately solves display on the variants page, storage etc. All that is needed is a phrase list for quick filling of the comments.

**Comment by moonso** [comment]
Yes @dnil that sounds like a clever solution. I guess that would be easy to implement, right @robinandeer ?

**Comment by dnil** [comment]
More phrases / list options from the CMMS/ClinGen lunch meeting:
- Not in disease related transcript
- Inherited from unaffected - AD 
- No plausible compound - AR
- Technical issues - likely false positive
- Not likely to affect protein function

Seems we need still would need a "dismiss" flag too. Should be activated by selecting any of the mentioned phrases/options. Grays out/clearly visibly marks dismissed variants as uninteresting on the variants view.

**Comment by dnil** [comment]
- Always c., p. coords, any comments, .

Too common in public databases
- frequency (ExAC/GnomAD, 1000G, swefreq?) 
Too common in local database
- locusdb (arch & current)
Not relevant phenotype
- OMIM entries
Inconsistent inheritance pattern
- OMIM inheritance pattern, Variant inheritance models, GT call
No compound variant
- Compound table
No plausible compound - AR
- Compounds table
Not in RefSeq transcript
- Transcripts table
Does not affect splicing
- SPIDEX (and eventually any other splice predictors)
Not in disease related transcript
- Transcripts table, color coding from Transcript overview
Inherited from unaffected - AD
- GT calls & pedigree
Technical issues - likely false positive
- GT calls (esp qual) & ideally pileup.js.. 
Not likely to affect protein function
[used also for variants like polyQ-expansions]
- Severity & Conservation, possilby pileup.
